attest/untrusted/src/sim.rs

```diff
@@ -249,26 +245,4 @@ mod test {
         let result = TcbInfo::try_from(&leaf_cert);
         assert!(result.is_ok());
     }
```

Comment:
Originally I had the test for `DcapVerifier` in this `sim.rs` file to prevent an module meant for the enclave from depending on a file outside of the enclave, but that's only for tests, so I think it's ok and it's easier to see when `DcapVerifier` is being tested where it's defined